knowledge-base/slider-max-parameter-zero-error.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Requires the Max parameter error in Slider
+description: Max parameter of zero throws an error
+type: troubleshooting
+page_title: Requires the Max parameter error in Slider
+slug: slider-kb-max-parameter-zero-error
+position: 
+tags: 
+ticketid: 1541425
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Slider for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+I'm using the Telerik Slider, and I noticed that I can't set the Max to 0. When I set the Max parameter to zero, the component throws an error.
```

Comment:
"noticed" doesn't contribute to customer context, as a person will not search by this word.

Consider quoting the exact error message below too:

```suggestion
I'm using the Telerik Slider, and I can't set Max to 0. When I set the Max parameter to zero, the component throws an error.
```